subprojects/docs/src/main/resources/header.html

```diff
@@ -200,6 +200,7 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                     <li><a href="../userguide/custom_plugins.html">Starting Plugin Development</a></li>
                     <li><a href="../userguide/designing_gradle_plugins.html">Designing Plugins</a></li>
                     <li><a href="../userguide/implementing_gradle_plugins.html">Implementing Plugins</a></li>
+                    <li><a href="../userguide/writing_in_kotlin.html">Writing Plugins in Kotlin</a></li>
```

Comment:
Maybe just rely in the link in "implementing_gradle_plugins", not put one here.

Reply:
I thought every page had an entry in the left column? Is it not the case?